### ISSUE TITLE
Can't Delete items

### ISSUE THREAD

**Issue by Mattwalsh129** [ghing/learning-collaborative-maps#44]
You can't delete programs. Like for North-Grand you can't delete it even though there is a double

**Comment by ghing** [comment]
Yeah, this seems like basic functionality that we should add.  This isn't too hard.
